shared/common-adapters/list2.desktop.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+import React, {PureComponent} from 'react'
+import AutoSizer from 'react-virtualized-auto-sizer'
+import * as Styles from '../styles'
+import {FixedSizeList, VariableSizeList} from 'react-window'
+import type {Props} from './list2'
+
+class List2 extends PureComponent<Props<any>, void> {
+  _keyExtractor = index => {
+    const item = this.props.items[index]
+    if (this.props.indexAsKey || !item) {
+      return String(index)
+    }
+
+    const keyProp = this.props.keyProperty || 'key'
+    return item[keyProp]
+  }
+
+  _fixed = ({height, width}) => (
+    <FixedSizeList
+      style={Styles.collapseStyles([styles.list, this.props.style])}
+      height={height}
+      width={width}
+      itemCount={this.props.items.length}
+      itemKey={this._keyExtractor}
+      itemSize={this.props.itemHeight}
+    >
+      {({index, style}) => <div style={style}>{this.props.renderItem(index, this.props.items[index])}</div>}
+    </FixedSizeList>
+  )
+
+  // $FlowIssue doesn't know itemHeight can't be number here -- we only call this when it's a function.
+  _variableItemSize = index => this.props.itemHeight(index, this.props.items[index])
```

Comment:
if you made this._variable = ({height, width, itemHeight}) (aka call from render) then flow would know its a valid function